docs/Sensor/SeeedStudio_XIAO/SeeedStudio_XIAO_ESP32S3/Application/Edgeimpulse/Image_classification_project.md

```diff
@@ -167,12 +178,18 @@ When the Build button is selected, a Zip file will be created and downloaded to
 
 Before we use the downloaded library, we need to enable the **ESP NN** Accelerator. For that, you can download a preliminary version from the [project GitHub](https://github.com/Mjrovai/XIAO-ESP32S3-Sense/blob/main/ESP-NN.zip), unzip it, and replace the ESP NN folder with it under:`src/edge-impulse-sdk/porting/espressif/ESP-NN`, in your Arduino library folder.
 
-<div style={{textAlign:'center'}}><img src="https://files.seeedstudio.com/wiki/xiaoesp32s3_kws/10.png" style={{width:800, height:'auto'}}/></div>
+:::warning Attention
 
-Copy the below inference code and paste to your Arduino IDE. 
+- Remember to replace the ESP-NN folder, do not have an existing folder or change its name to restore it. simply remove it.
+- The order of the header file could affect the compilation, so make sure to keep the correct order of the header files.
 
```

Comment:
We can remove this part as ESP32 Board definition includes in the ESP-NN library inbuilt now, no need to add externally.

Reply:
wow I didn't notice that.

Reply:
Thanks! Plus, is esp32 3.x compatible with Edge Impulse exported libraries?